pyproject.toml

```diff
@@ -16,6 +16,7 @@ build-backend = "poetry.core.masonry.api"
 [tool.poetry]
 packages = [
   {include = "sdx", from="src"},
+  {include = "research", from="."},
```

Comment:
not really sure if we should add it as part of the package.
but let's keep it for now.
thanks

Reply:
or maybe it could be added as a dependency just for development .. but we can take a look into that later